Code/Common/include/sitkCreateInterpolator.hxx

```diff
@@ -184,6 +187,16 @@ CreateInterpolator(const TImageType *image, InterpolatorEnum itype) {
     p->SetAlpha(1.0);
     return RType(p);
   }
+  case sitkLabelLinear: {
+#ifdef SITK_GENERIC_LABEL_INTERPOLATOR
+    using InterpolatorType = itk::LabelImageGenericInterpolateImageFunction<TImageType, itk::LinearInterpolateImageFunction, double>;
```

Comment:
Does ANTs make available other interpolator functions beside Linear for use with this type?

Reply:
just linear: https://github.com/ANTsX/ANTs/blob/46ab4e7c161151732624b8662be8023999dbfe78/Examples/make_interpolator_snip.tmpl

Reply:
but `genericlabel[linear]` apparently generally works well: https://github.com/ANTsX/ANTs/issues/677

Reply:
The original paper that introduced genericlabel showed linear was the best choice as well